robottelo/ui/locators.py

```diff
@@ -1383,6 +1383,15 @@ def __iter__(self):
     "repo.fetch_upstream": (
         By.XPATH, ("//form[@bst-edit-text='repository.docker_upstream_name']"
                    "/div[@class='bst-edit']/div/span[2]")),
+    "repo.fetch_packages": (
+        By.XPATH, "//td[span[text()='Packages']]/following-sibling::td",
```

Comment:
How about we fix this upstream as well, by providing a PR that would give us a unique ID? We've done it in the past (ask @oshtaier for more info).